DPB_Converger_config.ini

```diff
@@ -0,0 +1,29 @@
+[SOFTWARE]
```

Comment:
rename file to "DPB_converger_config.ini" with small "c" in converger. Furthermore, the abbreviation DPB is not intuitive -> rename *.ini and *.py file to swift_parameter_converger